src/main/java/frc/robot/commands/FlyWheelCommands/ShootNoteCommand.java

```diff
@@ -13,6 +14,12 @@ public ShootNoteCommand(FlyWheelSubsystem flyWheel){
     @Override
     public void execute(){
         flyWheel.rampUp();
+        isRampedUp = true;
+    }
+
+    @Override
+    public boolean isFinished(){
+        return isRampedUp;
```

Comment:
Same comment here as above - this will finish immediately

Reply:
cool, will fiz